ui/app/mainui/AppMain.qml

```diff
@@ -256,7 +256,7 @@ Item {
     Audio {
         id: sendMessageSound
         store: rootStore
-        track: Qt.resolvedUrl("../imports/assets/audio/send_message.wav")
+        source: "qrc:/imports/assets/audio/send_message.wav"
```

Comment:
Suggestion: common function similar to 'Style.svg' for importing wav files

Reply:
Except we have WAV but also one MP3 here :)